module/src/components/calendarInput/calendarInput.basic.scss

```diff
@@ -1,6 +1,7 @@
 .arm-calendar-select {
   margin-right: 0;
   flex: 1;
+  min-width: 57px;
```

Comment:
Magic number? If theres a reason for it should probably have a comment explaning why

Reply:
it's been put in a variable and documented in storybook